config/initializers/configuration.rb

```diff
@@ -12,7 +12,8 @@
   'slotMinutes' => 15,
   'dragOpacity' => 0.5,
   'selectable'  => true,
-  'timeFormat'  => "h:mm t{ - h:mm t}"
+  'timeFormat'  => "h:mm t{ - h:mm t}",
+  'defaultColor' => '#3B91AD'
```

Comment:
because all other variables are getting used on frontend, but if you just noticed at Line no: 18/19 we are using snake_case, as `mount_path` and `default_color` will be ruby realted variables.

I will think for a better implementation later.